addons/auth_signup/controllers/main.py

```diff
@@ -25,6 +25,18 @@ def web_login(self, *args, **kw):
             return request.redirect(request.params.get('redirect'))
         return response
 
+    @http.route('/web/redirect', type='http', auth='public', website=True, sitemap=False)
+    def web_redirect(self, token=False, db=False, **kw):
```

Comment:
I concur that the namespace and naming for this is not great.


Also `token` and `db` should default to `None`, `False` is an Odoo-ism for xmlrpc compatibility but that is not a concern here.